cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -459,7 +552,10 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 			if outMember.Type.IsAssignable() {
 				sw.Do("out.$.name$ = $.outType|raw$(in.$.name$)\n", args)
 			} else {
-				if g.convertibleOnlyWithinPackage(m.Type, outMember.Type) {
+				if g.preexists(m.Type, outMember.Type) {
+					args["function"] = g.preexisting[conversionType{m.Type, outMember.Type}]
```

Comment:
I'm vaguely concerned that a stray reference to function will catch an old value, since args may be reused. Not sure if it's worth fixing or not.